types/luxon/src/datetime.d.ts

```diff
@@ -171,947 +162,1086 @@ export interface ExplainedFormat {
-     * @param [options] - options that affect the creation of the Duration
-     * @param [options.conversionAccuracy='casual'] - the conversion system to use
+     * Returns the epoch milliseconds of this DateTime.
      */
-    diffNow(unit?: DurationUnits, options?: DiffOptions): Duration;
+    toMillis(): number;
 
     /**
-     * "Set" this DateTime to the end (meaning the last millisecond) of a unit of time
-     * @example
-     * DateTime.local(2014, 3, 3).endOf('month').toISO(); //=> '2014-03-31T23:59:59.999-05:00'
-     * @example
-     * DateTime.local(2014, 3, 3).endOf('year').toISO(); //=> '2014-12-31T23:59:59.999-05:00'
-     * @example
-     * DateTime.local(2014, 3, 3).endOf('week').toISO(); // => '2014-03-09T23:59:59.999-05:00', weeks start on Mondays
-     * @example
-     * DateTime.local(2014, 3, 3, 5, 30).endOf('day').toISO(); //=> '2014-03-03T23:59:59.999-05:00'
-     * @example
-     * DateTime.local(2014, 3, 3, 5, 30).endOf('hour').toISO(); //=> '2014-03-03T05:59:59.999-05:00'
+     * Returns the epoch seconds of this DateTime.
      */
-    endOf(unit: DurationUnit): DateTime;
+    toSeconds(): number;
 
     /**
-     * Equality check
-     * Two DateTimes are equal if they represent the same millisecond, have the same zone and location, and are both valid.
-     * To compare just the millisecond values, use `+dt1 === +dt2`.
+     * Returns an ISO 8601 representation of this DateTime appropriate for use in JSON.
      */
-    equals(other: DateTime): boolean;
+    toJSON(): string;
 
     /**
-     * Get the value of unit.
-     * @example
-     * DateTime.local(2017, 7, 4).get('month'); //=> 7
-     * @example
-     * DateTime.local(2017, 7, 4).get('day'); //=> 4
+     * Returns a BSON serializable equivalent to this DateTime.
      */
-    get(unit: keyof DateTime): number;
+    toBSON(): Date;
 
     /**
-     * Return whether this DateTime is in the same unit of time as another DateTime.
-     * Higher-order units must also be identical for this function to return `true`.
-     * Note that time zones are **ignored** in this comparison, which compares the **local** calendar time. Use {@link setZone} to convert one of the dates if needed.
-     * @example
-     * DateTime.now().hasSame(otherDT, 'day'); //~> true if otherDT is in the same current calendar day
+     * Returns a JavaScript object with this DateTime's year, month, day, and so on.
+     *
+     * @param opts - options for generating the object
+     * @param opts.includeConfig - include configuration attributes in the output. Default false
+     *
+     * @example DateTime.now().toObject() //=> { year: 2017, month: 4, day: 22, hour: 20, minute: 49, second: 42, millisecond: 268 }
      */
-    hasSame(other: DateTime, unit: DurationUnit): boolean;
+    toObject(options?: {
+        /**
+         * Include configuration attributes in the output
+         * @defaultValue false
+         */
+        includeConfig?: boolean;
```

Comment:
is this supposed to be there?:
https://moment.github.io/luxon/#/upgrading?id=other-breaking-changes
> DateTime#toObject no longer accepts an includeConfig option

Reply:
I believe it is a mistake on the upgrading guide.
includeConfig is still accepted in 2.0.1 and setting it true changes the object returned.
It is also in the API docs for DateTime. See https://moment.github.io/luxon/api-docs/index.html#datetimetoobject